arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -383,18 +391,20 @@ public virtual async Task<T> GetDocumentAsync<T>(string documentId)
         /// <param name="collectionName"></param>
         /// <param name="selectors"></param>
         /// <param name="query"></param>
+        /// <param name="serializationOptions"></param>
         /// <returns></returns>
         public virtual async Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>(
             string collectionName,
             IList<string> selectors,
-            DeleteDocumentsQuery query = null)
+            DeleteDocumentsQuery query = null,
+            ApiClientSerializationOptions serializationOptions = null)
```

Comment:
There's no need to provide for serializer options here, the `IList<string>` will be unaffected